presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -67,16 +67,16 @@ Date and Time Functions
 
 .. function:: from_unixtime(unixtime) -> timestamp
 
-    Returns the UNIX timestamp ``unixtime`` as a timestamp.
+    Returns the UNIX timestamp in seconds ``unixtime`` as a timestamp.
```

Comment:
The new sentence sounds strange. How about renaming the parameter `unixtime` to `seconds`? That way, the description would indicate it is a UNIX timestamp, and the unit would also be obvious.

Reply:
That's a good option, though many people do pass `unixtime` correctly because they see that as a good context clue for passing in seconds. Instead of adding "in seconds" or changing the parameter, maybe adding an additional sentence like " `unixtime` must be in seconds" would be sufficient.

Reply:
This is what I was thinking:
 ```
.. function:: from_unixtime(seconds) -> timestamp

    Returns the UNIX timestamp ``seconds`` as a timestamp.
```
But it also looks strange now that I see it written out. Adding an additional sentence sounds good. Something like

```
A UNIX timestamp is the number of seconds since ``1970-01-01 00:00:00``.

Reply:
It seems like modifying the existing structure just makes for an odd sentence. I updated the PR to reflect a second sentence and removed the odd `in seconds` from the original PR.